code/data.py

```diff
@@ -0,0 +1,80 @@
+'''
+Provide processed data ready for consumption by tensorflow model
+Return data always as numpy array
```

Comment:
Does the data need to be stored as numpy arrays? You are having memory problems, right? I think we should only need numpy arrays when it comes time to feed the placeholders, so maybe it makes sense to store them as list of lists until then.